charts/app/values.yaml

```diff
@@ -14,7 +14,7 @@ args: []
 
 replicas: 1
 buildId:
-serviceAccountName: "app" # infra should always create this service account
+serviceAccountName: ~  # infra should always create service account "app"
```

Comment:
Note: This might be a significant breaking change.

Reply:
Reverted the default service account and disabled the install step as the two are not compatible.